proto/zitadel/management.proto

```diff
@@ -841,6 +841,76 @@ service ManagementService {
         };
     }
 
+    // Sets a org metadata by key
+    rpc SetOrgMetadata(SetOrgMetadataRequest) returns (SetOrgMetadataResponse) {
+        option (google.api.http) = {
+            post: "/orgs/{id}/metadata/{key}"
```

Comment:
ID should not be possible to send for the request. Also in all the following requests

Should be either @adlerhurst what do you think is better? (org domains we have with orgs/me) :
post: "/orgs/me/metadata/{key}"
or
post: "/metadata/{key}"

Reply:
but that means you have to have a user in every org you want to set metadata right?
or is there another way how you can set metadata for an org for which you are not in, but have permission?